MVP/README.md

```diff
@@ -0,0 +1,76 @@
+# Platform-CI MVP
+
+## What's MVP?
+
+Minimal Viable Product (MVP) is a minimized set of Jenkins job definition (and supporting files) on Platform CI:
+
+  - for End-to-End workflow changes (batch update) via **Template / Default YAML**
+  - for create/update/scale tests (by individuals) via **Test / Job YAML**
+
+## How to create a CI job with MVP?
+
+1. ####Get MVP files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/MVP
+
+1. ####Modify [sample_job.yaml](/MVP/sample_job.yaml/) with your own tests *(skip this step if you just want a taste of creating job)*
+
+  - `ownership`
+
+     Replace those names with actual owner/co-owners of your test.
+     They will receive Jenkins email notifications accordingly.
+
+  - `project` - `name`/`component`
+
+     **name** is unique as identification of the job set.
+     And job trigger will watch brew builds for **component**.
+
+  - `shell`
+
+     This is THE key part of your job.
+     Fill in shell commands you've been using for submitting automation jobs in Beaker.
+
+     E.g. download/clone test metadata from team repo and run `bkr workflow-tomorrow` or `bkr job-submit`
+
+     (Keep the last two lines `bkr job-watch` and `bkr job-results`. They'll collect test results when jobs end.)
+
+1. ####Create or update your jobs
+
+  - Create/Update your jobs to Platform Jenkins master (Finally!)
+
+     >$ ./jenkins-jobs.sh update
+
+  - Want a dry-run before creating jobs? (validate your job definition)
+
+     >$ ./jenkins-jobs.sh test
+
+1. ####Done
+
```

Comment:
I am not sure what the idea is here. Where will the MVP component owners maintain this yaml ? In this github or their own repo ?